services/src/main/java/org/apache/druid/cli/CliIndexer.java

```diff
@@ -130,6 +130,7 @@ public void configure(Binder binder)
             JsonConfigProvider.bind(binder, "druid", DruidNode.class, Parent.class);
             JsonConfigProvider.bind(binder, "druid.worker", WorkerConfig.class);
 
+            CliPeon.configureIntermediaryData(binder);
```

Comment:
Are there any integration tests that verify shuffle with indexers continue to work after this change? I haven't looked at the existing integration tests closely

Reply:
There are many ITs that run indexers with the new config unset (which basically fallback to using "local" storage for storing intermediary segments via `LocalIntermediaryDataManager`). Specifically, `input source integration test with Indexer` runs ingestion with Hashed partitioning and maxNumConcurrentSubTasks=10, which would run the ingestion in two phases (first phase which persist to local using `LocalIntermediaryDataManager` and second phase which reads segments from first phase). Similarly, there are also some other ITs in compaction/auto compaction that uses Hashed partitioning.